src/explore-education-statistics-admin/src/pages/release/ReleasePageContainer.tsx

```diff
@@ -150,7 +150,12 @@ const ReleasePageContainer = ({
             </div>
           </div>
 
-          <Tag>{getReleaseStatusLabel(releasePublicationStatus.status)}</Tag>
+          <Tag>
+            {getReleaseStatusLabel(
+              releasePublicationStatus.status,
+              releasePublicationStatus.live,
```

Comment:
Have you tested this? There's another tag that also renders 'Live', so we'd end up with two 'Live' tags.